Framework/Algorithms/src/ReflectometryReductionOne2.cpp

```diff
@@ -709,7 +709,7 @@ void ReflectometryReductionOne2::findDetectorGroups() {
   // Sort the groups by the first spectrum number in the group (to give the same
   // output order as GroupDetectors)
   std::sort(m_detectorGroups.begin(), m_detectorGroups.end(),
-            [](const std::vector<size_t> a, const std::vector<size_t> b) {
+            [](const std::vector<size_t> &a, const std::vector<size_t> &b) {
```

Comment:
@gemmaguest thought this might be worth pointing out to you in case of possible existing UB

Reply:
In this case this is just a performance related thing, we were previously taking the entire group copying it, comparing two elements, then discarding the copy for every iteration of sort.

Reply:
Ah okay, think I got confused with a previous case where I had `&foo` instead of `&&foo`.